File: CS/1012 - Area.cs
using System;

class URI
{
    static void Main()
    {
        string[] numbers = Console.ReadLine().Split();

        float a = float.Parse(numbers[0]);
        float b = float.Parse(numbers[1]);
        float c = float.Parse(numbers[2]);
    
        float tri = a*c / 2;
        double cir = (c*c) * 3.14159;
        float tra = (a+b) * c / 2;
        float qua = (b*b);
        float ret = a*b;

        Console.WriteLine("TRIANGULO: {0}", tri.ToString("0.000"));
        Console.WriteLine("CIRCULO: {0}", cir.ToString("0.000"));
        Console.WriteLine("TRAPEZIO: {0}", tra.ToString("0.000"));
        Console.WriteLine("QUADRADO: {0}", qua.ToString("0.000"));
        Console.WriteLine("RETANGULO: {0}", ret.ToString("0.000"));
    }
}

File: PHP/1012 - Area.php
<?php

    $numbers = (string)readline();
    $numbers = explode(" ", $numbers);

    $a = (float)$numbers[0];
    $b = (float)$numbers[1];
    $c = (float)$numbers[2];

    $tri = ($a * $c) / 2;
    $cir = pow($c, 2) * 3.14159;
    $tra = ($a + $b) * $c / 2;
    $qua = pow($b, 2);
    $ret = $a * $b;

    $tri = number_format($tri, 3, ".", "");
    $cir = number_format($cir, 3, ".", "");
    $tra = number_format($tra, 3, ".", "");
    $qua = number_format($qua, 3, ".", "");
    $ret = number_format($ret, 3, ".", "");

    echo "TRIANGULO: $tri\n";
    echo "CIRCULO: $cir\n";
    echo "TRAPEZIO: $tra\n";
    echo "QUADRADO: $qua\n";
    echo "RETANGULO: $ret\n";
?>